api/v1beta1/foundationdbbackup_types.go

```diff
@@ -253,6 +272,11 @@ func (backup *FoundationDBBackup) BackupURL() string {
 		return backup.Spec.BlobStoreConfiguration.getURL(backup.BackupName(), backup.Bucket())
 	}
 
+	// mpatou: should we use file:// ?
+	if backup.Spec.FSConfiguration != nil {
+		return fmt.Sprintf("%s/%s", backup.Spec.FSConfiguration.URL, backup.BackupName())
```

Comment:
if this is a filesystem path, you should use
```suggestion
		return path.Join(backup.Spec.FSConfiguration.URL, backup.BackupName())
```

Reply:
The documentation of `fdb` indicates that you can use `file://` I don't think it would work ?